resources-for-container/files/env-runner/backend_services_data_importer.py

```diff
@@ -6,15 +8,15 @@ class BackendServicesDataImporter:
 
     def __init__(self, env: Environment):
         self.env = env
+        self.test_data_dump_filepath: str = self.env.mount_directory + "/test_data.sql"
```

Comment:
Minor suggestion: you might want to look at `os.path.join` instead of concatenating strings (https://docs.python.org/3/library/os.path.html#os.path.join). It is generally preferred because it works for both Windows and Unix style paths.

Reply:
Absolutely, thanks for flagging. I've actually used that in other parts of the codebase but forgot here. Going to fix this.

Reply:
This has been fixed here https://github.com/alphagov/digitalmarketplace-contained/pull/27/commits/cd3f6ce028bfa7fbabf0f08872047b94c93af643